### DISCUSSION TITLE
Error happens when changing Default branch in Safari

### DISCUSSION THREAD

**Discussion by QuqqU** [community/community#12790] [Other Feature Feedback, Questions, & Ideas]
p.s. I have no idea where to report this bug, anyone who knows where i have to do, please let me know.

- Safari version : 15.3(17612.4.9.1.8)
- Mac OS 12.2.1

When changing Default branch in Safari, error occurs like below.
It doesn't happen in Chrome.

![image](https://user-images.githubusercontent.com/30044929/158073281-2bd20d37-b3af-4e47-a820-2ae5dfa9a416.png)

**Comment by mariush2** [comment]
Same issue for me as well in Safari:
![image](https://user-images.githubusercontent.com/24507952/158163579-a6f3c76d-477c-4254-9adb-9a57df8e5403.png)
